standards/out-of-hours.md

```diff
@@ -0,0 +1,61 @@
+- The MOJ's ability to provide critical, time-sensitive justice services
+- Impact to staff's and suppliers' ability to do carry out their job safely and to the best of their ability
+- Impairing citizens', suppliers', and partners' ability to interact with MOJ services in a way that meets their needs
+
+These kinds of risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (realistically this means at least six people, ideally eight), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
+
+Either kind of person, and all of those on the spectrum between, however, can find it so stressful that they withdraw from the rota (leaving the service unsupported) unless a few core criteria are not met:
+
+- No one person is on call 50% of the time or more, and ideally no more than 30% of the time. This, along with the next criteria, mean that you will need at least six people, and ideally eight, to run a rota sustainably.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
+- If your service regularly has problems out of hours, or problems typically take multiple hours to resolve, you will need to take that into account when designing your rota. These sorts of situations will require more people, shorter shifts, or more layers of support.
```

Comment:
Do we want to add something about "if your service regularly has problems out of hours, your incident retrospectives should be ensuring that appropriate learning and investment is happening to improve that."?

Reply:
I've added exactly that. Thanks!